utils/findLinks.js

```diff
@@ -0,0 +1,64 @@
+
+const chalk = require('chalk')
+
+
+const findLinks = (datos, urlF) => {
+    let allthelink = /\[(.*)\]\S((http|https):\/\/[^\s\n)]+)(?=\))/gim;
+    let regularL = /(http|https):\/\/.*\)?/gi;
+    let textReg = /\[(.*)\]/gim
+    let objectLinks = [];
+    let newFile = process.argv.indexOf('--file');
+    let uri = process.argv[newFile + 1];
+    let joinArray = []
+    let linkAndtext;
+    let joinObject;
+    return new Promise((resolve) => {
+        let noStr = datos;
+        let str = noStr.toString();
+        let myArray;
+        while ((myArray = allthelink.exec(str)) !== null) {
+            joinArray.push(myArray[0]);
+        }
+        linkAndtext = joinArray;
+        var unique = linkAndtext.filter(onlyUnique)
+        //console.log(unique)
+        if (unique.length >= 1) {
+            unique.forEach(obj => {
+                let myLinks = obj.match(regularL);
+                let myText = obj.match(textReg);
```

Comment:
recomiendo definir mejor la respuesta del forEach para volverlo más clara
```suggestion
            unique.forEach(url => {
                let myLinks = url.match(regularL);
                let myText = url.match(textReg);
```